sites/partners/src/listings/PaperListingForm/index.tsx

```diff
@@ -268,6 +274,38 @@ const formatFormData = (
 
     delete unit.tempId
   })
+  summaries.forEach((summary) => {
+    summary.minOccupancy = toNumberOrNull(summary.minOccupancy)
+    summary.maxOccupancy = toNumberOrNull(summary.maxOccupancy)
+    summary.floorMin = toNumberOrNull(summary.floorMin)
+    summary.floorMax = toNumberOrNull(summary.floorMax)
+    summary.totalAvailable = toNumberOrNull(summary.totalAvailable)
+    summary.totalCount = toNumberOrNull(summary.totalCount)
```

Comment:
I'm not sure I understand why you're re-casting these on the same object? Can you not check the form once submit is hit?